internal/preferences.go

```diff
@@ -11,12 +11,14 @@ type InMemoryPreferences struct {
 	values          map[string]interface{}
 	lock            sync.RWMutex
 	changeListeners []func()
+	wg              *sync.WaitGroup
 }
 
 // Declare conformity with Preferences interface
 var _ fyne.Preferences = (*InMemoryPreferences)(nil)
 
 // AddChangeListener allows code to be notified when some preferences change. This will fire on any update.
+// listener should not try to write values
```

Comment:
This line added to the docs is not formatted like a complete sentence.

Reply:
what would be a better sentence?
the line is meant to indicate that the function `listener` should not try to write to `InMemoryPreferences`

Reply:
Maybe I mis-read it, in which case it's just missing the '.'.
I would imagine that it could be slightly easier to read by starting with "The passed 'listener'".